docs/source/AdministratorGuide/Systems/ResourceStatus/install.rst

```diff
@@ -126,15 +120,23 @@ Activate RSS
 
 If you did not see any problem, activate RSS by setting the CS option::
 
-    /Operations/Defaults/ResourceStatus/Config/State = Active
+    /Operations/[Defaults|SetupName]/ResourceStatus/Config/State = Active
 
 ------
 Agents
 ------
 
-There are few agents that are required:
+The agents that are required:
 
+    - CacheFeederAgent
     - SummarizeLogsAgent
 
-Please, install them and make sure they are up and running. Old agents, like the
-*InspectorAgent(s)* can be safely removed.
+The following agents are also necessary, but they won't do nothing until some policies are defined in the CS.
+The policy definitions is explained in :ref:`rss_advanced_configuration` ::
+
+    - ElementInspectorAgent
+    - SiteInspectorAgent
+    - TokenAgent
+    - EmailAgent
+
+Please, install them and make sure they are up and running.
```

Comment:
You could add a link to the  CodeDocumentation with the agent options https://dirac.readthedocs.io/en/latest/CodeDocumentation/ResourceStatusSystem/Agent/Agent_Module.html

Reply:
To link to the RSS agents overview page
```
:mod:`ResourceStatusSystem.Agent`
:mod:`Text for link <ResourceStatusSystem.Agent>`
```
Or to link to the individual agent modules, e.g.:
```
:mod:`~DIRAC.ResourceStatusSystem.Agent.SummarizeLogsAgent`
```

`:mod:` refers to the fact that it is a python module

The target is what is written in rst source code pages as (auto)module
https://dirac.readthedocs.io/en/latest/_sources/CodeDocumentation/ResourceStatusSystem/Agent/Agent_Module.rst.txt
https://dirac.readthedocs.io/en/latest/_sources/CodeDocumentation/ResourceStatusSystem/Agent/SiteInspectorAgent.rst.txt

`~DIRAC...` means only take the last part (i.e. SummarizeLogsAgent` as the text of the link)

There is some inconsistency in what the modules are called in the rst source creation :(